plugins/mewbot-discord_dice_roller/setup.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: 2021 - 2023 Mewbot Developers <mewbot@quicksilver.london>
+#
+# SPDX-License-Identifier: BSD-2-Clause
+
+"""
+Note - this is NOT intended to be used for local installation or as part of development.
+
+This is intended to make this module available via pypi
+It _should_ also work for local installation, but this functionality is not tested.
+"""
+
+raise NotImplementedError("Still under development.")
```

Comment:
Do we want this implemented for the purpose of this demo? or is it dependant on other things?

Reply:
It's...a little complicated. We're already doing some pretty nonstandard things with `setup.py` in other places with the multi-module build system. Probably could make it work right now, but I'm happy to ignore it.

I'm actually going to suggest (when we preoperly get there) using `pyproject.toml` rather than `setup.py` for the plugins -- that uses more sensible defaults and means there isn't a random python file for installation.

Reply:
I'm gonna have a think and a poke some, and see what I come up with.

Reply:
I have something which works, but feel it's outside the scope of this pull request.